tests/microbenchmarks.py

```diff
@@ -179,6 +186,7 @@ def test_loops(test_case):
                         pass
     """), timed=True)
 
+
 def test_dict_get(test_case):
     print("Running", "test_dictionary_get")
     test_case.runAsJava(adjust("""
```

Comment:
At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils